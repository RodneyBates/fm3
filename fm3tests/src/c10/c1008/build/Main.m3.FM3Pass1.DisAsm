 ItkEOF()
 ItkRightEnd()
 ItkModuleLtPatch(C4,L2(16_2),[11,0])
 ItkModuleBeginPatch(C735,L2(16_2),[33,2])
 ItkModuleRt(L2(16_2),[11,0])
 ItkDeclScopeRt(L1(16_1))
 ItkOpenScopeRt(L1(16_1))
 ItkStmtListRt(L0(16_0),[34,2])
 ItkStmtListLt(L0(16_0),[34,2])
 ItkOpenDeclListLtPatch(C10,L8(16_8),[13,2])
 ItkOpenDeclListRt(L8(16_8),[13,2])
 ItkSkipLtPatch(C673,L6(16_6))
 ItkSkipRt(L6(16_6))
 ItkDeclValAbsent([33,0])
 ItkVarDeclValue([33,0])
 ItkDeclTypeAbsent([33,0])
 ItkVarDeclType([30,4])
 ItkVarDeclIdListLtPatch(C673,L1(16_1),[30,4])
 ItkVarDeclIdListRt(L1(16_1),[30,4])
 ItkDeclId(L10(16_a),Id12"V5",L-9223372036854775808(16_8000000000000000),[30,4])
 ItkOpenDeclListSepPatch(C611,L4(16_4),[29,2])
 ItkVarDeclLtPatch(C611,[29,4])
 ItkVarDeclRt([29,4])
 ItkReservedIdRef(Rid"FALSE",[29,10])
 ItkVarDeclValue([29,7])
 ItkDeclTypeAbsent([29,7])
 ItkVarDeclType([29,4])
 ItkVarDeclIdListLtPatch(C611,L1(16_1),[29,4])
 ItkVarDeclIdListRt(L1(16_1),[29,4])
 ItkDeclId(L10(16_a),Id11"V4",L-9223372036854775808(16_8000000000000000),[29,4])
 ItkOpenDeclListSepPatch(C547,L3(16_3),[28,2])
 ItkVarDeclLtPatch(C547,[28,4])
 ItkVarDeclRt([28,4])
 ItkReservedIdRef(Rid"TRUE",[28,20])
 ItkVarDeclValue([28,7])
 ItkReservedIdRef(Rid"BOOLEAN",[28,9])
 ItkVarDeclType([28,4])
 ItkVarDeclIdListLtPatch(C547,L1(16_1),[28,4])
 ItkVarDeclIdListRt(L1(16_1),[28,4])
 ItkDeclId(L10(16_a),Id10"V3",L-9223372036854775808(16_8000000000000000),[28,4])
 ItkOpenDeclListSepPatch(C485,L2(16_2),[25,2])
 ItkVarDeclLtPatch(C485,[26,4])
 ItkVarDeclRt([26,4])
 ItkDeclValAbsent([28,2])
 ItkVarDeclValue([26,7])
 ItkReservedIdRef(Rid"BOOLEAN",[26,9])
 ItkVarDeclType([26,4])
 ItkVarDeclIdListLtPatch(C485,L1(16_1),[26,4])
 ItkVarDeclIdListRt(L1(16_1),[26,4])
 ItkDeclId(L10(16_a),Id9"U1",L-9223372036854775808(16_8000000000000000),[26,4])
 ItkSkipLtPatch(C442,L5(16_5))
 ItkSkipRt(L5(16_5))
 ItkDeclValAbsent([26,2])
 ItkVarDeclValue([25,7])
 ItkReservedIdRef(Rid"CHAR",[25,9])
 ItkVarDeclType([25,4])
 ItkVarDeclIdListLtPatch(C442,L0(16_0),[25,4])
 ItkVarDeclIdListRt(L0(16_0),[25,4])
 ItkDuplDeclId(Id7"V1",[25,4])
 ItkOpenDeclListSepPatch(C380,L1(16_1),[23,2])
 ItkVarDeclLtPatch(C380,[24,4])
 ItkVarDeclRt([24,4])
 ItkDeclValAbsent([25,2])
 ItkVarDeclValue([24,7])
 ItkReservedIdRef(Rid"REAL",[24,9])
 ItkVarDeclType([24,4])
 ItkVarDeclIdListLtPatch(C380,L1(16_1),[24,4])
 ItkVarDeclIdListRt(L1(16_1),[24,4])
 ItkDeclId(L10(16_a),Id8"V2",L-9223372036854775808(16_8000000000000000),[24,4])
 ItkSkipLtPatch(C343,L4(16_4))
 ItkSkipRt(L4(16_4))
 ItkDeclValAbsent([24,2])
 ItkVarDeclValue([23,12])
 ItkIdRefAtom(Id2"T",[23,14])
 ItkVarDeclType([23,4])
 ItkVarDeclIdListLtPatch(C343,L0(16_0),[23,4])
 ItkVarDeclIdListRt(L0(16_0),[23,4])
 ItkVarDeclLtPatch(C289,[22,4])
 ItkVarDeclRt([22,4])
 ItkDeclValAbsent([23,2])
 ItkVarDeclValue([22,7])
 ItkIdRefAtom(Id2"T",[22,9])
 ItkVarDeclType([22,4])
 ItkVarDeclIdListLtPatch(C289,L1(16_1),[22,4])
 ItkVarDeclIdListRt(L1(16_1),[22,4])
 ItkDeclId(L10(16_a),Id7"V1",L-9223372036854775808(16_8000000000000000),[22,4])
 ItkOpenDeclListSep(L3(16_3),[13,2])
 ItkVarDeclLtPatch(C229,[19,6])
 ItkVarDeclRt([19,6])
 ItkDeclValAbsent([21,0])
 ItkVarDeclValue([19,8])
 ItkReservedIdRef(Rid"BOOLEAN",[19,10])
 ItkVarDeclType([19,6])
 ItkVarDeclIdListLtPatch(C229,L1(16_1),[19,6])
 ItkVarDeclIdListRt(L1(16_1),[19,6])
 ItkDeclId(L10(16_a),Id6"U",L-9223372036854775808(16_8000000000000000),[19,6])
 ItkOpenDeclListSep(L2(16_2),[13,2])
 ItkSkipLtPatch(C181,L3(16_3))
 ItkSkipRt(L3(16_3))
 ItkDeclValAbsent([19,0])
 ItkVarDeclValue([18,9])
 ItkReservedIdRef(Rid"CHAR",[18,11])
 ItkVarDeclType([18,6])
 ItkVarDeclIdListLtPatch(C181,L0(16_0),[18,6])
 ItkVarDeclIdListRt(L0(16_0),[18,6])
 ItkDuplDeclId(Id4"V0",[18,6])
 ItkOpenDeclListSep(L2(16_2),[13,2])
 ItkVarDeclLtPatch(C121,[17,6])
 ItkVarDeclRt([17,6])
 ItkDeclValAbsent([18,0])
 ItkVarDeclValue([17,8])
 ItkReservedIdRef(Rid"REAL",[17,10])
 ItkVarDeclType([17,6])
 ItkVarDeclIdListLtPatch(C121,L1(16_1),[17,6])
 ItkVarDeclIdListRt(L1(16_1),[17,6])
 ItkDeclId(L10(16_a),Id5"V",L-9223372036854775808(16_8000000000000000),[17,6])
 ItkOpenDeclListSep(L1(16_1),[13,2])
 ItkSkipLtPatch(C79,L2(16_2))
 ItkSkipRt(L2(16_2))
 ItkDeclValAbsent([17,0])
 ItkVarDeclValue([16,14])
 ItkIdRefAtom(Id2"T",[16,16])
 ItkVarDeclType([16,6])
 ItkVarDeclIdListLtPatch(C79,L0(16_0),[16,6])
 ItkVarDeclIdListRt(L0(16_0),[16,6])
 ItkOpenDeclListSep(L1(16_1),[13,2])
 ItkVarDeclLtPatch(C22,[15,6])
 ItkVarDeclRt([15,6])
 ItkDeclValAbsent([16,0])
 ItkVarDeclValue([15,9])
 ItkIdRefAtom(Id2"T",[15,11])
 ItkVarDeclType([15,6])
 ItkVarDeclIdListLtPatch(C22,L1(16_1),[15,6])
 ItkVarDeclIdListRt(L1(16_1),[15,6])
 ItkDeclId(L10(16_a),Id4"V0",L-9223372036854775808(16_8000000000000000),[15,6])
 ItkSkipLtPatch(C10,L1(16_1))
 ItkSkipRt(L1(16_1))
 ItkIdRefAtom(Id3"LONG",[13,11])
 ItkOpenScopeLt(L1(16_1))
 ItkDeclScopeLt(L1(16_1))
 ItkLeftEnd()
 ItkBOF()
     0
