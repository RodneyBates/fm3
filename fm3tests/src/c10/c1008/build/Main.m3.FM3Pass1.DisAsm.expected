 ItkEOF()
 ItkRightEnd()
 ItkModuleLtPatch(C4,L1(16_1),[11,0])
 ItkModuleBeginPatch(C757,L1(16_1),[33,2])
 ItkModuleRt(L1(16_1),[11,0])
 ItkDeclScopeRt(L1(16_1))
 ItkOpenScopeRt(L1(16_1))
 ItkStmtListRt(L0(16_0),[34,2])
 ItkStmtListLt(L0(16_0),[34,2])
 ItkOpenDeclListLtPatch(C10,L9(16_9),[13,2])
 ItkOpenDeclListRt(L9(16_9),[13,2])
 ItkSkipLtPatch(C722,L5(16_5))
 ItkSkipRt(L5(16_5))
 ItkDeclValAbsent([33,0])
 ItkVarDeclValue([33,0])
 ItkDeclTypeAbsent([33,0])
 ItkVarDeclType([33,0])
 ItkVarDeclIdListLtPatch(C695,L1(16_1),[30,4])
 ItkVarDeclIdListRt(L1(16_1),[30,4])
 ItkDeclId(L10(16_a),Id12"V5",L-9223372036854775808(16_8000000000000000),[30,4])
 ItkOpenDeclListSepPatch(C633,L4(16_4),[29,2])
 ItkVarDeclLtPatch(C633,[29,4])
 ItkVarDeclRt([29,4])
 ItkReservedIdRef(Rid"FALSE",[29,10])
 ItkVarDeclValue([29,7])
 ItkDeclTypeAbsent([29,7])
 ItkVarDeclType([29,7])
 ItkVarDeclIdListLtPatch(C633,L1(16_1),[29,4])
 ItkVarDeclIdListRt(L1(16_1),[29,4])
 ItkDeclId(L10(16_a),Id11"V4",L-9223372036854775808(16_8000000000000000),[29,4])
 ItkOpenDeclListSepPatch(C569,L3(16_3),[28,2])
 ItkVarDeclLtPatch(C569,[28,4])
 ItkVarDeclRt([28,4])
 ItkReservedIdRef(Rid"TRUE",[28,20])
 ItkVarDeclValue([28,17])
 ItkReservedIdRef(Rid"BOOLEAN",[28,9])
 ItkVarDeclType([28,7])
 ItkVarDeclIdListLtPatch(C569,L1(16_1),[28,4])
 ItkVarDeclIdListRt(L1(16_1),[28,4])
 ItkDeclId(L10(16_a),Id10"V3",L-9223372036854775808(16_8000000000000000),[28,4])
 ItkOpenDeclListSepPatch(C507,L2(16_2),[25,2])
 ItkVarDeclLtPatch(C507,[26,4])
 ItkVarDeclRt([26,4])
 ItkDeclValAbsent([28,2])
 ItkVarDeclValue([28,2])
 ItkReservedIdRef(Rid"BOOLEAN",[26,9])
 ItkVarDeclType([26,7])
 ItkVarDeclIdListLtPatch(C507,L1(16_1),[26,4])
 ItkVarDeclIdListRt(L1(16_1),[26,4])
 ItkDeclId(L10(16_a),Id9"U1",L-9223372036854775808(16_8000000000000000),[26,4])
 ItkSkipLtPatch(C481,L4(16_4))
 ItkSkipRt(L4(16_4))
 ItkDeclValAbsent([26,2])
 ItkVarDeclValue([26,2])
 ItkReservedIdRef(Rid"CHAR",[25,9])
 ItkVarDeclType([25,7])
 ItkVarDeclIdListLtPatch(C463,L0(16_0),[25,4])
 ItkVarDeclIdListRt(L0(16_0),[25,4])
 ItkDuplDeclId(Id7"V1",L1(16_1),[25,4])
 ItkOpenDeclListSepPatch(C401,L1(16_1),[23,2])
 ItkVarDeclLtPatch(C401,[24,4])
 ItkVarDeclRt([24,4])
 ItkDeclValAbsent([25,2])
 ItkVarDeclValue([25,2])
 ItkReservedIdRef(Rid"REAL",[24,9])
 ItkVarDeclType([24,7])
 ItkVarDeclIdListLtPatch(C401,L1(16_1),[24,4])
 ItkVarDeclIdListRt(L1(16_1),[24,4])
 ItkDeclId(L10(16_a),Id8"V2",L-9223372036854775808(16_8000000000000000),[24,4])
 ItkSkipLtPatch(C376,L3(16_3))
 ItkSkipRt(L3(16_3))
 ItkDeclValAbsent([24,2])
 ItkVarDeclValue([24,2])
 ItkIdRefAtom(Id2"T",[23,14])
 ItkVarDeclType([23,12])
 ItkVarDeclIdListLtPatch(C364,L0(16_0),[23,4])
 ItkVarDeclIdListRt(L0(16_0),[23,4])
 ItkVarDeclLtPatch(C310,[22,4])
 ItkVarDeclRt([22,4])
 ItkDeclValAbsent([23,2])
 ItkVarDeclValue([23,2])
 ItkIdRefAtom(Id2"T",[22,9])
 ItkVarDeclType([22,7])
 ItkVarDeclIdListLtPatch(C310,L1(16_1),[22,4])
 ItkVarDeclIdListRt(L1(16_1),[22,4])
 ItkDeclId(L10(16_a),Id7"V1",L-9223372036854775808(16_8000000000000000),[22,4])
 ItkOpenDeclListSep(L4(16_4),[13,2])
 ItkVarDeclLtPatch(C250,[19,6])
 ItkVarDeclRt([19,6])
 ItkDeclValAbsent([21,0])
 ItkVarDeclValue([21,0])
 ItkReservedIdRef(Rid"BOOLEAN",[19,10])
 ItkVarDeclType([19,8])
 ItkVarDeclIdListLtPatch(C250,L1(16_1),[19,6])
 ItkVarDeclIdListRt(L1(16_1),[19,6])
 ItkDeclId(L10(16_a),Id6"U",L-9223372036854775808(16_8000000000000000),[19,6])
 ItkOpenDeclListSep(L3(16_3),[13,2])
 ItkSkipLtPatch(C219,L2(16_2))
 ItkSkipRt(L2(16_2))
 ItkDeclValAbsent([19,0])
 ItkVarDeclValue([19,0])
 ItkReservedIdRef(Rid"CHAR",[18,11])
 ItkVarDeclType([18,9])
 ItkVarDeclIdListLtPatch(C201,L0(16_0),[18,6])
 ItkVarDeclIdListRt(L0(16_0),[18,6])
 ItkDuplDeclId(Id4"V0",L1(16_1),[18,6])
 ItkOpenDeclListSep(L3(16_3),[13,2])
 ItkVarDeclLtPatch(C141,[17,6])
 ItkVarDeclRt([17,6])
 ItkDeclValAbsent([18,0])
 ItkVarDeclValue([18,0])
 ItkReservedIdRef(Rid"REAL",[17,10])
 ItkVarDeclType([17,8])
 ItkVarDeclIdListLtPatch(C141,L1(16_1),[17,6])
 ItkVarDeclIdListRt(L1(16_1),[17,6])
 ItkDeclId(L10(16_a),Id5"V",L-9223372036854775808(16_8000000000000000),[17,6])
 ItkOpenDeclListSep(L2(16_2),[13,2])
 ItkSkipLtPatch(C111,L1(16_1))
 ItkSkipRt(L1(16_1))
 ItkDeclValAbsent([17,0])
 ItkVarDeclValue([17,0])
 ItkIdRefAtom(Id2"T",[16,16])
 ItkVarDeclType([16,14])
 ItkVarDeclIdListLtPatch(C99,L0(16_0),[16,6])
 ItkVarDeclIdListRt(L0(16_0),[16,6])
 ItkOpenDeclListSep(L2(16_2),[13,2])
 ItkVarDeclLtPatch(C42,[15,6])
 ItkVarDeclRt([15,6])
 ItkDeclValAbsent([16,0])
 ItkVarDeclValue([16,0])
 ItkIdRefAtom(Id2"T",[15,11])
 ItkVarDeclType([15,9])
 ItkVarDeclIdListLtPatch(C42,L1(16_1),[15,6])
 ItkVarDeclIdListRt(L1(16_1),[15,6])
 ItkDeclId(L10(16_a),Id4"V0",L-9223372036854775808(16_8000000000000000),[15,6])
 ItkOpenDeclListSep(L1(16_1),[13,2])
 ItkTypeDeclLtPatch(C10,[13,7])
 ItkTypeDeclTypePatch(C18,[13,9])
 ItkTypeDeclRt([13,7])
 ItkIdRefAtom(Id3"LONG",[13,11])
 ItkDeclId(L8(16_8),Id2"T",L181(16_b5),[13,7])
 ItkOpenScopeLt(L1(16_1))
 ItkDeclScopeLt(L1(16_1))
 ItkLeftEnd()
 ItkBOF()
     0
